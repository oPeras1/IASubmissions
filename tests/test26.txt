1	2	2	2	2	3	4	3	3	3	5	5	5	6	6	7	7	8	8	8
1	1	2	2	3	3	4	4	4	3	5	9	6	6	7	7	10	8	11	11
1	12	12	3	3	3	3	3	3	3	9	9	6	6	6	10	10	8	11	11
12	12	12	12	12	13	13	3	13	3	3	9	9	14	6	10	10	10	15	11
16	12	12	17	12	13	13	13	13	13	18	19	14	14	6	6	6	20	15	11
16	17	17	17	21	21	21	21	21	18	18	19	19	14	14	22	20	20	15	15
16	23	23	23	23	23	21	21	21	24	18	19	14	14	25	22	20	26	26	15
16	23	23	23	23	23	27	27	27	24	18	19	14	14	25	22	20	26	26	15
28	28	28	29	29	27	27	24	24	24	18	18	18	25	25	22	22	22	26	26
28	30	30	29	29	29	29	24	24	24	24	18	18	25	31	22	22	22	31	26
30	30	30	30	32	32	33	34	34	34	35	35	18	31	31	31	31	31	31	31
30	36	36	32	32	32	33	33	34	35	35	35	35	35	35	31	37	37	37	37
38	36	36	36	36	32	33	33	34	39	40	40	40	41	35	42	42	37	42	42
38	38	38	38	43	32	32	32	39	39	39	40	41	41	41	42	42	42	42	44
38	45	38	43	43	43	32	32	39	39	39	40	40	40	41	41	41	44	44	44
45	45	45	45	43	46	32	32	39	47	48	48	48	49	49	41	44	44	50	44
51	51	51	43	43	46	32	47	47	47	48	48	48	52	49	49	50	50	50	50
51	51	51	51	43	46	46	47	53	47	47	52	52	52	54	54	54	50	55	50
51	56	56	51	51	46	46	53	53	47	52	52	52	54	54	54	54	50	55	55
51	51	56	56	51	51	51	53	47	47	47	54	54	54	57	57	57	57	57	55