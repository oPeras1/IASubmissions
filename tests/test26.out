T	2	T	T	T	3	L	3	3	3	L	L	L	6	6	S	S	L	L	L
T	T	2	T	3	3	L	L	L	3	L	9	6	6	S	S	10	L	11	T
T	12	L	I	I	I	I	3	3	3	S	S	6	L	L	10	T	8	T	T
L	L	L	12	12	13	13	3	T	3	3	S	S	14	L	T	T	T	15	T
I	12	12	L	12	13	13	T	T	T	18	I	14	14	L	6	6	20	S	11
I	L	L	L	I	I	I	I	21	18	18	I	19	S	S	22	L	L	S	S
I	23	23	23	23	23	21	21	21	I	18	I	S	S	25	22	L	26	26	S
I	23	I	I	I	I	L	L	L	I	18	I	14	14	S	22	L	26	S	15
L	L	L	29	T	27	L	24	24	I	T	T	T	S	S	T	T	T	S	S
L	30	30	T	T	T	29	24	24	I	24	T	18	S	31	22	T	22	31	S
I	I	I	I	32	32	L	T	T	T	S	S	18	I	I	I	I	31	31	31
30	36	L	32	32	32	L	33	T	35	35	S	S	35	35	31	T	T	T	37
T	36	L	L	L	32	L	L	34	T	L	L	40	T	35	L	42	T	42	42
T	T	38	38	T	32	S	32	T	T	39	L	T	T	T	L	L	L	42	T
T	45	38	T	T	T	S	S	39	T	39	L	40	40	41	41	41	44	T	T
I	I	I	I	43	L	32	S	39	L	T	T	T	S	S	41	44	44	50	T
51	51	51	43	43	L	32	L	L	L	48	T	48	52	S	S	I	I	I	I
51	51	51	51	43	L	L	47	S	47	47	S	S	52	54	54	54	50	S	50
51	S	S	51	L	46	46	S	S	47	S	S	52	L	54	54	54	50	S	S
51	51	S	S	L	L	L	S	47	47	47	L	L	L	I	I	I	I	57	S